components/menu/menu-item-link.js

```diff
@@ -61,13 +71,12 @@ class MenuItemLink extends MenuItemMixin(LitElement) {
 	}
 
 	render() {
-		const rel = this.target ? 'noreferrer noopener' : undefined;
-		return html`
-			<a download="${ifDefined(this.download)}" href="${ifDefined(this.href)}" rel="${ifDefined(rel)}" target="${ifDefined(this.target)}" tabindex="-1">
```

Comment:
It looks like the `tabindex="-1"` is being omitted in the new version. This will result in the `d2l-menu-item-link`'s internal anchor being focusable via `[tab]`, which I think will cause a regression in the menu's focus behaviour. For all menu items, the host is the focusable, marked up with the appropriate Aria `role`, and `d2l-menu` manages the focus behaviour (arrow keys).

Note: one of the demo menus contains `d2l-menu-item-link`s, but they are the last items in the menu, making regression of this behaviour less evident.